cg/cli/workflow/balsamic/base.py

```diff
@@ -355,11 +314,7 @@ def run(context, dry, run_analysis, config_path, priority, email, case_id):
 
 @balsamic.command()
 @click.option(
-    "-d",
-    "--dry-run",
-    "dry_run",
-    is_flag=True,
-    help="print to console, " "without actualising",
+    "-d", "--dry-run", "dry_run", is_flag=True, help="print to console, " "without actualising"
```

Comment:
Why is help string split into two strings?

Reply:
I'll fix that
```suggestion
    "-d", "--dry-run", "dry_run", is_flag=True, help="print to console without actualising"
```